python/lsst/meas/deblender/deblend.py

```diff
@@ -842,8 +840,8 @@ def run(self, exposures, sources):
             created by the multiband templates.
             If `self.config.saveTemplates` is `False`, then this item will be None
         """
-        psfs = {B:exp.getPsf() for B, exp in exposures.items()}
-        return self.deblend(exposures, sources, psfs)
+        psfs = {f:mExposure[f].getPsf() for f in mExposure.filters}
```

Comment:
`filter`? `f` always makes me think of file handle. Although in a comprehension it is pretty obvious.

Reply:
Yeah, even if you'd like me to change it in the loops, I'd prefer to keep the `f` in the list comprehensions.